app/views/hiring_staff/vacancies/_expiry_time_field.html.haml

```diff
@@ -27,4 +27,4 @@
                   input_html: {class: 'govuk-input govuk-date-input__input govuk-input--width-2'},
                   required: true,
                   include_blank: '- -',
-                  input_html: {class: ("govuk-input--error" if @application_details_form.errors[:expiry_time].present?)}
\ No newline at end of file
+                  input_html: {class: ("govuk-input--error" if form.errors[:expiry_time].present?)}
```

Comment:
This input has two `input_haml` statements. 
There's one on line 27 and this one on line 30.
Should probably condense them.